src/it/projects/profile-with-deps-import/pom.xml

```diff
@@ -0,0 +1,42 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
probably needs copyright notice; run `mvn verify` to be sure
Maybe not since this is mojohaus, not apache

Reply:
this seems okay - we are adhering to other IT pom file headers